### PR TITLE
Improve documentation of URL formats for `pulumi login`

### PR DESCRIPTION
**Pull request by metral**
Related https://github.com/pulumi/pulumi/issues/3194

### CODE REVIEW

File: cmd/login.go [hunk 1]
```diff
@@ -47,12 +47,12 @@ func newLoginCmd() *cobra.Command {
...
 			"\n" +
-			"By default, this will log into `app.pulumi.com`. If you prefer to log into a separate instance\n" +
+			"By default, this will log into `api.pulumi.com`. If you prefer to log into a separate instance\n" +
 			"of the Pulumi service, such as Pulumi Enterprise, specify a URL. For example, run\n" +
 			"\n" +
-			"    $ pulumi login https://pulumi.acmecorp.com\n" +
+			"    $ pulumi login https://api.pulumi.acmecorp.com\n" +
 			"\n" +
-			"to log in to a Pulumi Enterprise server running at the pulumi.acmecorp.com domain.\n" +
+			"to log in to a Pulumi Enterprise server running at the api.pulumi.acmecorp.com and app.pulumi.acmecorp.com domains.\n" +
```
Comment (lukehoban) [inline cmd/login.go]: We may want to break this line manually so that it stays within the current line wrap lengths.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
+
+- Fix API URL to use with pulumi login
```
Comment (lukehoban) [inline CHANGELOG.md]:
```suggestion
- Improve documentation of URL formats for `pulumi login`
```

File: cmd/login.go (hunk 1 above)
Comment (metral) [reply cmd/login.go]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (metral) [reply CHANGELOG.md]: Done

**Pull request merged**